app/transport/http/user.go

```diff
@@ -15,7 +15,7 @@ func (uh UserHandler) CreateUser() http.Handler {
 		var user entities.User
 
 		if err := json.NewDecoder(req.Body).Decode(&user); err != nil {
-			WriteJSONResponse(w, http.StatusBadRequest, Response{MsgBadRequest}, uh.logger)
+			WriteJSONResponse(w, http.StatusBadRequest, Response{MsgBadRequest + err.Error()}, uh.logger)
```

Comment:
Instead of concatenating strings let's better add `Details` field in the `Response` structure.

Reply:
Thanks, added Details field